app/models/user.rb

```diff
@@ -111,8 +115,9 @@ def can_prefill_emis?
   end
 
   def can_access_id_card?
-    beta_enabled?(uuid, 'veteran_id_card') && loa3? && edipi.present? && veteran?
-  rescue # Default to false for any veteran_status error
+    beta_enabled?(uuid, 'veteran_id_card') && loa3? && edipi.present? &&
```

Comment:
not sure why my comment got suppressed. 

Performance NITPICK: 

```ruby
loa3? && edipi.present? && beta_enabled?(uuid, 'veteran_id_card') &&
```

will avoid invoking a database call unless necessary. LOA3 users will always fetch EDIPI one way or the other as part of users profile, and future hits will be a redis cache hit. Whereas for LOA1 you'd effectively be hitting the database first, every time.